makefile

```diff
@@ -370,7 +371,33 @@ swaggo-build:
 swaggo-render:
 	npx @redocly/cli build-docs $(buildDir)/swagger.json -o $(buildDir)/redoc-static.html
 
-phony += swaggo swaggo-install swaggo-format swaggo-build swaggo-render
+
+# Variables
+# TODO: Update this after DEVPROD-14517 is complete
+OPENAPI_FWS_CONFIG_URL := https://foliage-web-services.cloud-build.staging.corp.mongodb.com/foliage_web_services.json
+OPENAPI_FWS_HARDCODED_CONFIG := thirdparty/clients/foliage_web_services.json
+OPENAPI_FWS_OUTPUT_DIR := thirdparty/clients/fws
+OPENAPI_FWS_CONFIG := packageName=fws,packageVersion=1.0.0,packageTitle=FoliageWebServices,packageDescription="Foliage Web Services",apiTests=false,modelTests=false
+OPENAPI_GENERATOR := bin/openapi-generator-cli.sh
+
+# Main rule for generating the client
+# TODO: Remove the hardcoded config rule once DEVPROD-14517 is complete
+# fws-client: download-fws-config generate-client
+fws-client: generate-fws-client
+
+download-fws-config:
+	@echo "Downloading OpenAPI config..."
+	curl -L -o openapi-config.json $(OPENAPI_FWS_CONFIG_URL)
```

Comment:
Can we download this into the bin directory rather than just leave it potentially in the top-level namespace?

Also, is the intended workflow that devs manually download the FWS config to update it in the future? Or will there be an automated process to do it? If it's manual, do we need to supply Kanopy OIDC auth to the request, or can it just be curled without auth?

Reply:
Thank you! That was the correct word I needed to search figure out how to auth against it.